apps/backend/src/app/auctions/dto/create-auction.dto.ts

```diff
@@ -1,4 +1,11 @@
+import { IsNumber, IsPositive, IsString } from 'class-validator';
+import { Category } from '../../categories/entities/category.entity';
+
 export class CreateAuctionDto {
+  @IsString()
   readonly title: string;
-  readonly category: string;
+
+  @IsNumber()
+  @IsPositive()
+  readonly category: Category;
```

Comment:
Are we able to check is given ID exists in database at this point? Just asking

Reply:
not in an easy way, I think

Reply:
Generally it is possible

Reply:
I added it :)